docs/topics/coroutine-context-and-dispatchers.md

```diff
@@ -521,8 +543,9 @@ For the demo, we launch ten coroutines that delay for a different time:
 ``` 
 
 In our main function we create the activity, call our test `doSomething` function, and destroy the activity after 500ms.
-This cancels all the coroutines that were launched from `doSomething`. We can see that because after the destruction 
-of the activity no more messages are printed, even if we wait a little longer.
+This cancels the coroutines that were launched from `doSomething`.
```

Comment:
"All" used to emphasize that *every coroutine* gets cancelled, which is an important point here.